### PR TITLE
Allow payments for all classifications

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-862

### Change description ###
Removed validation which allows payments for only new applications.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4621495&bid=15498045)

File: src/integrationTest/resources/zipcontents/supplementary_evidence_with_ocr/metadata.json [hunk 1]
```diff
@@ -22,5 +22,8 @@
     }
   ],
   "payments": [
+    {
+      "document_control_number": "1111002"
+    }
```
Comment (doncem) [inline src/integrationTest/resources/zipcontents/supplementary_evidence_with_ocr/metadata.json]: there should be a scenario with empty payments ;). is there one?

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java [hunk 2]
```diff
@@ -293,7 +293,6 @@ private ZipFileProcessingResult processZipFileContent(
             EnvelopeValidator.assertPaymentsEnabledForContainerIfPaymentsArePresent(
                 envelope, paymentsEnabled, containerMappings.getMappings()
             );
-            EnvelopeValidator.assertClassificationNewApplicationIfPaymentsArePresent(envelope);
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java]: ☺️

File: src/integrationTest/resources/zipcontents/supplementary_evidence_with_ocr/metadata.json (hunk 1 above)
Comment (aliveni) [reply src/integrationTest/resources/zipcontents/supplementary_evidence_with_ocr/metadata.json]: Added the test scenario

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/bulk-scan-processor/bulk-scan-processor/pullRequest?prid=4621495&bid=15498045)

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/bulk-scan-processor/pullRequest?prid=4621495&bid=15498045)

**Pull request merged**